CHANGES.md

```diff
@@ -0,0 +1,13 @@
+
+## 1.1.0 (2021-01-28)
+
+* Improve how the library declares API extensions (https://github.com/arturo-ai/arturo-stac-api/pull/54)
+* Add postgres bulk transactions client (https://github.com/arturo-ai/arturo-stac-api/pull/59)
+* Update TiTiler version (https://github.com/arturo-ai/arturo-stac-api/pull/61)
+* Use attrs instead of dataclasses (https://github.com/arturo-ai/arturo-stac-api/pull/73)
+* Remove postgres database connection from API layer (https://github.com/arturo-ai/arturo-stac-api/pull/74)
+* Fix `pre-commit` config (https://github.com/arturo-ai/arturo-stac-api/pull/75)
+
+## 1.0.0 (2020-09-25)
+
+* First PyPi release!
```

Comment:
I can revert to text if you don't like md

Reply:
no this is great!